analysis_options.yaml

```diff
@@ -32,184 +32,186 @@ analyzer:
 
 linter:
   rules:
-    always_specify_types: false
-    omit_local_variable_types: false # opposite of always_specify_types
-    # these rules are documented on and in the same order as
+    # These rules are documented on and in the same order as
     # the Dart Lint rules page to make maintenance easier
     # https://github.com/dart-lang/linter/blob/master/example/all.yaml
+
     always_declare_return_types: true
     always_put_control_body_on_new_line: true
+    use_string_buffers: false
     # - always_put_required_named_parameters_first # we prefer having parameters in the same order as fields https://github.com/flutter/flutter/issues/10219
     always_require_non_null_named_parameters: true
+    always_specify_types: false
     # - always_use_package_imports # we do this commonly
     annotate_overrides: true
     # - avoid_annotating_with_dynamic # conflicts with always_specify_types
-    # - avoid_as # required for implicit-casts: true
     avoid_bool_literals_in_conditional_expressions: true
     # - avoid_catches_without_on_clauses # we do this commonly
     # - avoid_catching_errors # we do this commonly
-#    - avoid_classes_with_only_static_members
+    avoid_classes_with_only_static_members: false # <---------- Gery enable and check this I think is a really good one.
```

Comment:
agree.

Reply:
I meant that once I finish all the corrections with the "finals" check if it is worth enable that lint.

Beacause AppColors is a Class with static colors and that is a ver heavy change.

Reply:
ah gotcha. 
hum, lets leave it false for now